src/test/java/uk/gov/hmcts/ccd/domain/service/caseaccess/CaseAccessOperationTest.java

```diff
@@ -1300,53 +1349,140 @@ void setUp() {
         @Test
         @DisplayName("should remove single case user role")
         void shouldRemoveSingleCaseUserRole() {
+
             // ARRANGE
+            when(applicationParams.getEnableAttributeBasedAccessControl()).thenReturn(false);
+
             List<CaseAssignedUserRoleWithOrganisation> caseUserRoles = Lists.newArrayList(
                 new CaseAssignedUserRoleWithOrganisation(CASE_REFERENCE.toString(), USER_ID, CASE_ROLE)
             );
 
             // for an existing relation and then after removal
-            when(caseUserRepository.findCaseUserRoles(
-                argThat(arg -> arg.contains(CASE_ID)),
-                argThat(arg -> arg.contains(USER_ID))
-            )).thenReturn(List.of(
-                createCaseUserEntity(CASE_ID, CASE_ROLE, USER_ID)
-            )).thenReturn(new ArrayList<>());
+            mockExistingCaseUserRoles(
+                // before
+                List.of(createCaseUserEntity(CASE_ID, CASE_ROLE, USER_ID)),
+                // after
+                new ArrayList<>()
+            );
 
             // ACT
             caseAccessOperation.removeCaseUserRoles(caseUserRoles);
 
             // ASSERT
             verify(caseUserRepository, times(1)).revokeAccess(CASE_ID, USER_ID, CASE_ROLE);
+
+            verifyNoInteractions(roleAssignmentService);
+        }
+
+        @Test
+        @DisplayName("RA set to true, should remove single case user role")
+        void shouldRemoveSingleCaseUserRoleForRA() {
+
+            // ARRANGE
```

Comment:
this is `Given, when, then` elsewhere - but this really is a minor comment !

Reply:
Resolved in 1ee8ce11700eb2cfa7b7763bbd00bd9846172417.

Sorry, I use them both interchangeably.  Looking at the history I introduced ARRANGE/ACT/ASSERT under RDM-8606 over a year ago and then recently added GIVEN/WHEN/THEN in RDM-10924, #1555.  As the later is in shorter supply (14 vs 50) I have re-worded the recent additions.